src/towncrier/create.py

```diff
@@ -56,14 +65,21 @@ def _main(ctx, directory, config, filename, edit, content):
     * .removal - a deprecation or removal of public API,
     * .misc - a ticket has been closed, but it is not of interest to users.
     """
-    return __main(ctx, directory, config, filename, edit, content)
+    __main(ctx, directory, config, filename, edit, content)
```

Comment:
I'm confused by both the change and why mypy isn't complaining, even if I add `warn_no_return = true`. What is happening her? Did they change their stance on implicit None returns?

Reply:
And now I'm confused by your confusion. ;) Why would mypy complain about it?

What happened is that when annotating `_main()` (single), I looked at what values `__main()` (double) can return and saw that it doesn't have a `return` statement. Therefore I thought it was cleaner to not return the implicit `None` but instead remove the `return` from `_main()` (single).

Reply:
`warn_no_return` is documented as "Shows errors for missing return statements on some execution paths.", but `__main()` (double) has no `return` statement on **any** execution path.

Reply:
ugh yeah you're entirely correct